plugins/linux-pulseaudio/pulse-wrapper.c

```diff
@@ -185,6 +185,28 @@ int_fast32_t pulse_get_source_info_list(pa_source_info_cb_t cb, void* userdata)
 	return 0;
 }
 
+int_fast32_t pulse_get_sink_info_list(pa_sink_info_cb_t cb, void* userdata)
```

Comment:
pulse_get_source_info (the function right below this one) uses `type_name *data_name`, so it's already inconsistent.  As I understand it from previous conversations with Jim, any additions/changes should follow the style guide.  You're welcome to change it or leave it.  I'm just making note of it.